src/adapters/sequelize.js

```diff
@@ -94,7 +94,7 @@ module.exports = (model, opts) => {
       });
 
       _.remove(fields, (field) =>
-        _.includes(fieldNamesToExclude, field.columnName));
+        _.includes(fieldNamesToExclude, field.columnName) && !field.isPrimaryKey);
```

Comment:
@GuillaumeCisco could you detail the reasons why you decided to remove this check here:
https://github.com/ForestAdmin/forest-express-sequelize/pull/598/files#r562012387

Reply:
If I remember correctly `isPrimaryKey` was not a real attribute, but `primaryKey` was.
Maybe it depends on the sequelize version...
We can add it back without any issues.

Reply:
It seems the regression was introduced in this PR https://github.com/ForestAdmin/forest-express-sequelize/pull/504/files#diff-c8a0ca2d518fd6bf783df5b68798d4d5d0c7de61aa40726052fa02b1672691ba 

It's not related to sequelize version. I'm not sure the fix I propose is the best (in october it was obvious this was a refacto error but since you decided to completely remove it I don't know if there is any other reason (according to the comment, it may simplify user experience somehow ?)) but at least it reintroduces the missing field in forest when a foreignKey belongs to self.